client/src/keep_alive/helpers.rs

```diff
@@ -0,0 +1,30 @@
+use selium_protocol::error_codes::REPLIER_ALREADY_BOUND;
+use selium_std::errors::{QuicError, Result, SeliumError};
+use std::{io, task::Poll};
+
+pub fn is_disconnect_error(err: &io::Error) -> bool {
+    matches!(
+        err.kind(),
+        io::ErrorKind::ConnectionReset | io::ErrorKind::NotConnected
+    )
+}
+
+pub fn is_bind_error(code: u32) -> bool {
+    matches!(code, REPLIER_ALREADY_BOUND)
+}
+
+pub fn is_recoverable_error(err: &SeliumError) -> bool {
```

Comment:
Would it be cleaner to just use a single match statement here?

Reply:
Yep! I was originally going to add more codes to match on, and forgot to clean up after that wasn't the case 👍 Fixed!